dandiapi/api/services/dandiset/__init__.py

```diff
@@ -0,0 +1,60 @@
+from typing import Tuple
+
+from django.core.exceptions import PermissionDenied, ValidationError
+from django.db import transaction
+
+from dandiapi.api.models.dandiset import Dandiset
+from dandiapi.api.models.version import Version
+from dandiapi.api.services.version.metadata import _normalize_version_metadata
+
+
+def create_dandiset(
+    *,
+    user,
+    identifier: int = None,
```

Comment:
```suggestion
    identifier: int | None = None,
```
Interesting that this doesn't fail type-checking. Pylance/pywright reports it as an error for me locally, but mypy doesn't catch it in CI....